### PR TITLE
Update KoLV language in v1.8 guide

### PR DESCRIPTION
**Pull request by jtcohen6**
Now that dbt Core v1.8 has its final release, update language to clarify that dbt Cloud customers should select "Keep on latest version" to get access to v1.8 functionality (and already more)

### CODE REVIEW

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## Keep on latest version 
 
-With dbt Cloud, you can get early access to many new features and functionality before they're in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) setting for more details.
+dbt Cloud is going "versionless." Going forward, you get early access to new features and functionality before they're available in final releases of dbt Core. Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development and deployment environments to get access to everything in dbt Core v1.8 and more.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]: hey @jtcohen6 , added 'staging' here too - is that ok?

```suggestion
dbt Cloud is going "versionless." This means you'll automatically get early access to new features and functionality before they're available in final releases of dbt Core. 

Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production environments [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
```

Comment (jtcohen6) [reply website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]: Good call @mirnawong1. I think this sentence should either read:
- "development and deployment environments"
- "development, staging, and production environments"

Given that the environment hierarchy is:
- type: development
- type: deployment
    - deployment_type: production
    - deployment_type: staging
    
That is, "production" and "staging" are subtypes of "deployment" (= for running jobs that deploy models on a schedule/trigger) as distinct from "development" (= operating on models interactively in response to user edits/commands)

Comment (mirnawong1) [reply website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]: sound good! i'll keep it as development, staging, and production environments to enhance the 'staging' aspect if that's cool!

**Review by mirnawong1 — APPROVED** [review]
lgtm !

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## Keep on latest version 
 
-With dbt Cloud, you can get early access to many new features and functionality before they're in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) setting for more details.
+dbt Cloud is going "versionless." This means you'll automatically get early access to new features and functionality before they're available in final releases of dbt Core. 
+
+Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production environments [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
```

**Pull request merged**